distribution/dashboard/ui/src/pages/Tenant/helper.ts

```diff
@@ -65,3 +66,26 @@ export function formatNewTenantForm(
 
   return result;
 }
+/**
+ * encrypt ossAccessId,ossAccessKey,bakEncryptionPassword
+ *
+ * format scheduleDates
+ */
+export function formatNewBackupForm(originFormData: any, publicKey: string) {
+  let formData = clone(originFormData);
+  if (formData.bakEncryptionPassword) {
+    formData.bakEncryptionPassword = encryptText(
+      originFormData.bakEncryptionPassword,
+      publicKey,
+    );
+  }
+  formData.ossAccessId = encryptText(originFormData.ossAccessId, publicKey);
```

Comment:
Is it unnecessary to use originFormData after cloning？

Reply:
I believe it's a good practice as it can help prevent any potential side effects.